modal/_container_entrypoint.py

```diff
@@ -816,6 +815,13 @@ def main(container_args: api_pb2.ContainerArguments, client: Client):
                 metadata: Message = container_app.object_handle_metadata[object_id]
                 obj._hydrate(object_id, _client, metadata)
 
+        # Initialize clustered functions.
+        if function_def._experimental_group_size > 1:
+            initialize_clustered_function(
+                client,
+                container_args.task_id,
+            )
```

Comment:
j/w but did you consider getting the cluster info lazily the first time user code asks for it rather than as part of the container startup? Not really sure if that's better not; I'm just not totally certain if this is the right place to be doing it.